cmd/account/mgmt/account-assign.go

```diff
@@ -192,8 +200,14 @@ func (o *accountAssignOptions) findUntaggedAccount(rootOu string) (string, error
 		}
 
 		if !isOwned {
-			accountAssignID = *a.Id
-			break
+			isSuspended, err := isSuspended(*a.Id, o.awsClient)
+			if err != nil {
+				return "", err
+			}
```

Comment:
If the account is suspended, would it make sense here to `continue` to the next loop to find an account that isn't suspended?

Reply:
Hmm.. sorry I'm not following. Won't this return at line 178, and stop this function from running?

Reply:
Ah duh, this only returns on err, not on isSuspended